client/src/amplify-ui/cli-feature-flag/feature-flag-summary.tsx

```diff
@@ -0,0 +1,54 @@
+import {Component, Host, h, Prop, Element} from "@stencil/core";
+import {FeatureFlag} from "./feature-flag";
+import {tableStyle, summaryCell} from "./feature-flag.style";
+
+@Component({tag: "amplify-feature-flag-summary", shadow: false})
+export class AmplifyFeatureFlagSummary {
+  @Element() el: HTMLElement;
+
+  /** name of the feature flag */
+  @Prop() readonly name: string;
+
+  /** data of the feature flag */
+  @Prop() readonly feature: FeatureFlag;
+
+  render() {
+    return (
+      <div>
+        <docs-in-page-link targetId={this.name}>
+          <h3 id={this.name}>{this.name}</h3>
+        </docs-in-page-link>
+
+        {this.feature.description ? (
+          <p>{this.feature.description}</p>
+        ) : (
+          undefined
+        )}
+        <table class={tableStyle}>
+          <thead>
+            <tr>
+              <th class={summaryCell}>Type</th>
```

Comment:
You could improve this CSS specification by adding a single class to `tr` and adding a rule for `th` elements to implement the style specified in `summaryCell`